content/pages/faq.md

```diff
@@ -0,0 +1,52 @@
+Title: FAQ
+Date: 2015-01-28 12:00
+Category: pages
+Summary: WiCS Frequenctly Asked Questions
+
+Here you will find answers to questions we commonly receive. If you have a 
+question not listed below, please do not hesitate to reach out to us!
+
+# Why do you sometimes hold women-only events? #
+
+An example of an event that's women-only is our mentoring program, where we 
+match up junior female students with more senior female students to talk 
+about their experiences and support them throughout their academic career. 
+Women can obviously seek out male mentors as well but they don't quite share 
+the same experiences so the impact of the mentoring is not the same. There 
+are also events that are traditionally very competitive (such as hackathons) 
+that women feel more comfortable coming out to if they're women-only. 
+
+Holding events which are only open to women helps some women deal with
+[exceptionalism](http://geekfeminism.wikia.com/wiki/Exceptionalism 
+"Geek Feminism Wiki: Exceptionalism") and 
+[stereotype threat](http://geekfeminism.wikia.com/wiki/Stereotype_threat
+"Geek Feminism Wiki: Stereotype Threat").
+
+Our women-only events are not publicly advertised and we always 
+**very explicitly** specify if an event is going to be women-only.
+
+
+# What does women-only mean? #
+
+Borrowing from Double Union's 
+[membership policy](http://www.doubleunion.org/membership 
+"Double Union: Membership"), by "women-only"  we mean if you identify as 
+women in some way which is significant to you (including trans, queer,
+cis, gender non-conforming, and etc ... ) 
+
+# How can I get involved? #
+
+Please see the Volunteer Opportunities section on our [About page](/about).
```

Comment:
The idiomatic Pelican way to write this link would be `[About page]({filename}/pages/about.md)`, in case we ever change the way we generate full page links (and also to accommodate production vs. testing settings) :)